docs/types.md

```diff
@@ -32,19 +32,17 @@ Docs: https://developer.mozilla.org/Add-ons/WebExtensions
 
 ## Dictionaries
 
-* Validate `install.rdf`
-* Deal with any other dictionary specific rules.
+* Deal with any dictionary specific rules.
 
 Docs: https://developer.mozilla.org/docs/Creating_a_spell_check_dictionary_add-on
 
 ## Lang Packs
 
-* Validate `install.rdf`
 * Validate `chrome.manifest`
 * Deal with any other langpack specific rules.
```

Comment:
Hmm, this page urgently needs a cleanup… we don't really lint dictionaries and don't have any support for `chrome.manifest` anymore (which was legacy as well). Doesn't have to be happening here, or rather, it shouldn't and we can open another cleanup issue for docs specifically.